pkg/ccrm/ccrm_appinst.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2024 EdgeXR, Inc
```

Comment:
Since this is the implementation of `AppInstPlatformAPI`, does it make sense to call the files `ccrm_appinst_api.go` to be consistent with all the implementations of the APIs defined in `.proto` files? Same question applies to cloudlet and clusterinst

Reply:
We could, although I don't think we need to. Everything in here is for the APIs anyway.